src/test/java/com/exactpro/junit5/TestFormatter.java

```diff
@@ -0,0 +1,66 @@
+package com.exactpro.junit5;
+
+import io.netty.buffer.ByteBuf;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TestFormatter {
+
+    @Test
+    void testNameOneByte() throws IOException {
+        String hex = "12";
+        String methodName = new Object() {
+        }
+            .getClass()
+            .getEnclosingMethod()
+            .getName();
+
+        List<ByteBuf> arguments = new ArrayList<>();
+        arguments.add(ByteBufUtils.fromHex(hex));
+        WithByteBufInvocationContext invocationContext =
+            new WithByteBufInvocationContext(methodName, new WithByteBufMethodContext(), arguments);
+        assertEquals(methodName + " [ " + hex + " ]", invocationContext.getDisplayName(1));
+    }
+
+    @Test
+    void testNameMultipleByteOneBuffer() throws IOException {
+        String hex = "12 13 a4";
+        String methodName = new Object() {
+        }
+            .getClass()
+            .getEnclosingMethod()
+            .getName();
+
+        List<ByteBuf> arguments = new ArrayList<>();
+        arguments.add(ByteBufUtils.fromHex(hex));
+        WithByteBufInvocationContext invocationContext =
+            new WithByteBufInvocationContext(methodName, new WithByteBufMethodContext(), arguments);
+        assertEquals(methodName + " [ " + hex + " ]", invocationContext.getDisplayName(1));
+    }
+
+    @Test
+    void testNameMultipleByteMultipleBuffer() throws IOException {
+        String[] hexes = {"12 13", "a4", "11"};
+        String methodName = new Object() {
+        }
+            .getClass()
+            .getEnclosingMethod()
+            .getName();
+
+        List<ByteBuf> arguments = new ArrayList<>();
+        for (String hex : hexes) {
+            arguments.add(ByteBufUtils.fromHex(hex));
+        }
+
+        WithByteBufInvocationContext invocationContext =
+            new WithByteBufInvocationContext(methodName, new WithByteBufMethodContext(), arguments);
+        assertEquals(methodName + " [ 12 13 ] [ a4 ] [ 11 ]", invocationContext.getDisplayName(1));
```

Comment:
Can we make it uppercase: "A4"?

Reply:
by default Integer.toHexString converts to lowercase letters, if you want I can add toUpperCase at the end

Reply:
There are 3 options:
- toUpperCase()
- String.format()
- `if (x < 10) '0' + x else 'A' + x`